src/argilla/client/client.py

```diff
@@ -636,13 +636,21 @@ def _load_records_internal(
         task = dataset.task
 
         task_config = {
-            TaskType.text_classification: (SdkTextClassificationRecord, DatasetForTextClassification),
-            TaskType.token_classification: (SdkTokenClassificationRecord, DatasetForTokenClassification),
-            TaskType.text2text: (SdkText2TextRecord, DatasetForText2Text),
+            TaskType.text_classification: (
+                SdkTextClassificationRecord,
+                DatasetForTextClassification,
+                {"id", "event_timestamp", "status", "inputs", "multi_label"},
```

Comment:
should this not also include the "text" parm? or will this be over-ruled by adding "inputs"?